Admin/PageAdmin.php

```diff
@@ -336,7 +336,7 @@ protected function configureFormFields(FormMapper $formMapper)
         if ($this->hasSubject() && !$this->getSubject()->getId()) {
             $formMapper
                 ->with('form_page.group_main_label')
-                    ->add('site', null, array('required' => true, 'read_only' => true))
+                    ->add('site', 'null', array('attr' => array('required' => 'required', 'readonly' => 'readonly')))
```

Comment:
Looking at other instances in the file, I can see you're right. It should be 'required' =>'true'